RC 82: Manuscript Cases and other changes

nimisha20 opened the pull request.

nimisha20 committed:
21904: Documentation on editing avatar animation json

nimisha20 committed:
Merge pull request #6 from highfidelity/master

Update with latest changes

nimisha20 committed:
Updating with master changes

nimisha20 committed:
Merge branch 'RC-82' of https://github.com/nimisha20/hifi-docs-sphinx into RC-82

nimisha20 committed:
22063: Document changes to bubble app in the docs

nimisha20 committed:
22043: Document New Audio Features

nimisha20 committed:
21849: WMR SteamVR Requirement not documented

jennaingersoll commented on docs/source/explore/get-started/adjust-settings.rst:
```diff
@@ -55,24 +55,41 @@ High Fidelity spatializes audio in real-time based on the location of the listen
 +----------------------------+--------------------------------------------------------------------------------------+
-| *Input Device Settings* >  | The audio level meter is visible on the top left corner of your screen. By default,  |
-| Show audio level meter     | this is always shown. You can uncheck this box to stop showing the meter.            |
+| Audio level meter          | The audio level meter is visible on the top left corner of your screen. By default,  |
```
By default, the audio level meter is visible on the top left corner of your screen. Uncheck this box to hide the meter.

nimisha20 committed:
21849: WMR SteamVR Requirement not documented

Minor correction to remove an image.

jennaingersoll commented on docs/source/explore/get-started/adjust-settings.rst:
```diff
@@ -55,24 +55,41 @@ High Fidelity spatializes audio in real-time based on the location of the listen
+|                            | this is always shown. You can uncheck this box to stop showing the meter.            |
++----------------------------+--------------------------------------------------------------------------------------+
+| Push To Talk (T)           | Enable or disable Push To Talk for other users to hear you when your microphone is   |
+|                            | muted. When enabled, press and hold ``T`` to talk in desktop mode and press both       |
```
I think there's an extra space on this line?

nimisha20 replied:
Yeah! Done.

jennaingersoll commented on docs/source/explore/get-started/adjust-settings.rst:
```diff
@@ -55,24 +55,41 @@ High Fidelity spatializes audio in real-time based on the location of the listen
+|                            | triggers on your controllers in VR mode. This feature works only when you are        |
+|                            | focused on the Interface window.                                                     |
++----------------------------+--------------------------------------------------------------------------------------+
+| Stereo input               | Enable or disable stereo input.                                                      |
```
Can we describe what stereo input is?

nimisha20 replied:
Done.

jennaingersoll commented on docs/source/explore/get-started/adjust-settings.rst:
```diff
@@ -55,24 +55,41 @@ High Fidelity spatializes audio in real-time based on the location of the listen
+| Environment volume         | Use the slider to increase or decrease the domain's volume.                          |
+|                            |                                                                                      |
++----------------------------+--------------------------------------------------------------------------------------+
+| System Sound volume        | Use the slider to increase or decrease your system's volume.                         |
```
Does this essentially reduce the volume of both people and environments?

nimisha20 replied:
Done.

nimisha20 replied:
No it doesn't. It is the UI volume (tablet clicks).

jennaingersoll commented on docs/source/explore/get-started/adjust-settings.rst:
```diff
@@ -55,24 +55,41 @@ High Fidelity spatializes audio in real-time based on the location of the listen
+| People volume              | Use the slider to increase or decrease the volume of other users in the domain.      |
+|                            |                                                                                      |
++----------------------------+--------------------------------------------------------------------------------------+
+| Environment volume         | Use the slider to increase or decrease the domain's volume.                          |
```
```suggestion
Use the slider to increase or decrease the volume of ambient sound in the domain's volume. For example, use this option to decrease the sound of background music or special effects like fireworks.
```

nimisha20 replied:
done

jennaingersoll commented on docs/source/explore/get-started/adjust-settings.rst:
```diff
@@ -55,24 +55,41 @@ High Fidelity spatializes audio in real-time based on the location of the listen
++----------------------------+--------------------------------------------------------------------------------------+
 
 ^^^^^^^^^^^^^^^^^^^^^^^
 Input Devices 
```
GitHub won't let me comment on anything that isn't a change you made. So I'll put it here...

Under "Output Devices", there's a grammar error: ("s" at the end of "drive") 

An audio output device produces a signal that drives another device’s input.

jennaingersoll replied:
Also - under 'Show Shadows', can we add "If you have a low game rate or have trouble loading, disabling this option may improve your performance."

jennaingersoll reviewed (commented):
I reviewed everything except the animation stuff, since you indicated its not ready yet :)

nimisha20 committed:
21802: Troubleshooting for people with old/missing HFC wallets

Removing old Wallet info.

nimisha20 committed:
21852: Doc Request: Information on our particle system and properties in the Create app.

nimisha20 committed:
Revert "21852: Doc Request: Information on our particle system and properties in the Create app."

This reverts commit 23f92afc973c1035bf639fc326351183ca178d9e.

nimisha20 committed:
21852: Doc Request: Information on our particle system and properties in the Create app.

nimisha20 committed:
22043: Document New Audio Features (Implemented review feedback)

nimisha20 committed:
22030: Locomotion Changes for RC82

nimisha20 committed:
Locomotion Changes for RC82

nimisha20 committed:
Merge pull request #7 from highfidelity/master

Updating to the latest version of the master branch

nimisha20 committed:
22030: Locomotion Changes for RC82 - Added HTML tables for better formatting

nimisha20 committed:
Merge branch 'RC-82' of https://github.com/nimisha20/hifi-docs-sphinx into RC-82

nimisha20 committed:
22050: Docs: Provide user education on DDV

nimisha20 committed:
21201: Docs Request: baked lighting from a uv2 channel

nimisha20 committed:
21904: Documentation on editing avatar animation json

nimisha20 committed:
21852: Doc Request: Information on our particle system and properties in the Create app. (latest version)

nimisha20 committed:
22030: Locomotion Changes for RC82

nimisha20 committed:
22050 (minor correction)

nimisha20 committed:
8391: Docker docs

jennaingersoll commented on docs/source/explore/get-started/desktop.rst:
```diff
@@ -13,7 +13,7 @@ Desktop users are restricted to using their keyboard and mouse to do things in H
 
-.. image:: _images/controls-keyboard.png
+.. image:: _images/controls-keyboard.jpg
 
```
Under movement, Shift + Arrow key also strafes (when enabled). Pretty sure this is new.

nimisha20 replied:
Done.

nimisha20 committed:
21234: Document Domain Settings for Large Groups

nimisha20 committed:
21651: Avatar Entity documentation

nimisha20 committed:
8391 Docker docs (Minor corrections)

nimisha20 committed:
22274: Shape app no longer exists, remove from docs

jennaingersoll commented on docs/source/create/3d-models/pbr-materials-guide.rst:
```diff
@@ -74,6 +74,7 @@ Notes:
 1. If you set transparency with a texture, the transparency (alpha) should be in the material’s albedo texture, as a PNG file with transparency and not as a separate transparency texture.
 2. High Fidelity’s renderer can draw two different kinds of transparency: "alpha" (255 graduates steps of transparency, no shading on surface, casts no shadows,) and "mask" (binary transparency, full shading of opaque surface, whole surface casts shadow.)
 3. To determine whether a texture is treated as a mask or as alpha, the engine looks for alpha values between 2% and 98%. An easy way to create a mask texture is to save your image as a PNG-8 since it only supports binary transparency, while PNG-24 supports a range of transparency levels.
+4. Currently, we support using a second UV set with the following texture channels only - Light map and Ambient Occlusion. 
```
I want to remove anything that suggests we'll "fix" or change it in the future. So...

```suggestion
4. To add a second UV set, use either the Light Map or Ambient Occlusion texture channels. ```

nimisha20 replied:
I've changed it a bit to keep the terminology of using the second UV set.

jennaingersoll commented on docs/source/create/avatars/custom-animations.rst:
```diff
@@ -2,48 +2,74 @@
 #####################################
 
-Avatars in High Fidelity use the built in standard set of animations by default. These animations are generic and do not change based on your avatar. You can have your avatar express customized animations by replacing the existing ones.
+Your avatar can express custom animations such as dancing, juggling, or waving. You can do this by replacing or overriding the standard animations. All avatars in High Fidelity use the built in standard set of animations by default. These animations are generic and do not change based on your avatar. 
```
```suggestion
You can express yourself by overriding High Fidelity's standard set of animations with your own custom animations such as dancing, juggling, or waving. Any custom animations you set up will be independent to each avatar you own and wear.
```

jennaingersoll commented on docs/source/create/avatars/custom-animations.rst:
```diff
@@ -2,48 +2,74 @@
+Your avatar can express custom animations such as dancing, juggling, or waving. You can do this by replacing or overriding the standard animations. All avatars in High Fidelity use the built in standard set of animations by default. These animations are generic and do not change based on your avatar. 
 
-.. note:: The methods detailed below are being updated and changed frequently to create a better process to import custom animations for avatars. Keep this in mind before customizing your avatar's animations.
+.. note:: We often update our process to import custom animations for avatars, to make it more user friendly. Keep this in mind before customizing your avatar's animations.
```
```suggestion
.. note:: We often update our process for importing custom animations to make it more user friendly. As you develop custom animations, keep in mind that you may need to modify them in the future as our custom animation support continues to improve.
```

jennaingersoll commented on docs/source/create/avatars/custom-animations.rst:
```diff
@@ -2,48 +2,74 @@
+Prerequisites
+----------------------------------
+
+Consider familiarizing yourself with these terms.
```
```suggestion
As we delve deeper into creating custom animations, we may use terminology that you are unfamiliar with. Here are some terms you might come across:
```

jennaingersoll commented on docs/source/create/avatars/custom-animations.rst:
```diff
@@ -2,48 +2,74 @@
-You can enrich your High Fidelity experience by making your avatar express customized animations. Do this by replacing the standard animations for your avatar. 
-
-We've listed the different ways you can do this:
+You can have your avatar express custom animations in different ways:
```
```suggestion
You can have your avatar use your custom animations by overriding the default animations. There are two different ways to do this:
```

jennaingersoll commented on docs/source/create/avatars/custom-animations.rst:
```diff
@@ -65,16 +91,12 @@ This method uses :doc:`scripting <../../script>` and the `MyAvatar <https://apid
 2. You can replace the animation for each role with a custom animation (FBX file) using `MyAvatar.overrideRoleAnimation <https://apidocs.highfidelity.com/MyAvatar.html#.overrideRoleAnimation>`_.
 
-We've listed the animation roles and their description. Animation roles are frequently updated. We recommend using ``MyAvatar.getAnimationRoles`` to get the latest animation roles being used. The standard animation FBX files for these roles can be found in the High Fidelity source code repository on `GitHub <https://github.com/highfidelity/hifi/tree/master/interface/resources/avatar/animations>`_ ...
+We've listed the animation roles and their description. These are frequently updated. We recommend using ``MyAvatar.getAnimationRoles`` to get the latest animation roles being used. The standard animation FBX files for these roles can be found in the High Fidelity source code repository on `GitHub <https://github.com/highfidelity/hifi/tree/master/interface/resources/avatar/animations>`_.
```
```suggestion
We've listed the animation roles and their description. These are frequently updated, so we recommend using ``MyAvatar.getAnimationRoles`` to get the latest animation roles before continuing. The standard animation FBX files for these roles can be found in the High Fidelity source code repository on `GitHub <https://github.com/highfidelity/hifi/tree/master/interface/resources/avatar/animations>`_.
```

jennaingersoll commented on docs/source/create/avatars/custom-animations.rst:
```diff
@@ -171,26 +193,24 @@ We've listed the animation roles and their description. Animation roles are freq
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-When you wear an avatar in High Fidelity, the animation system must blend and layer a series of animations from FBX files as well as perform Inverse Kinematics on the joints to best match the head and hand sensors. 
+You can edit or replace the existing Avatar Animation JSON file to override the standard animations. 
```
```suggestion
If you're not comfortable using a script, you can edit or replace the existing Avatar Animation JSON file to override the standard animations. 
```

jennaingersoll commented on docs/source/create/avatars/custom-animations.rst:
```diff
@@ -171,26 +193,24 @@ We've listed the animation roles and their description. Animation roles are freq
 2. In Interface, pull up your HUD or Tablet and go to **Avatar**.
 3. Click on the Settings icon on the top-right corner. 
-4. Under 'Avatar animation JSON', add the URL for your Animation Graph  file.  ![](avatar-settings.png)
+4. Under 'Avatar animation JSON', paste the URL for your JSON file. 
```
```suggestion
4. Under 'Avatar Animation JSON', paste the URL for your JSON file. 
```

jennaingersoll commented on docs/source/create/avatars/custom-animations.rst:
```diff
@@ -171,26 +193,24 @@ We've listed the animation roles and their description. Animation roles are freq
-4. Under 'Avatar animation JSON', add the URL for your Animation Graph  file.  ![](avatar-settings.png)
+4. Under 'Avatar animation JSON', paste the URL for your JSON file. 
+
+.. image:: _images/avatar-anim.png
```
I don't think this image is needed.

nimisha20 replied:
Removing it.

jennaingersoll commented on docs/source/create/avatars/custom-animations.rst:
```diff
@@ -207,13 +227,14 @@ Examples
 ^^^^^^^^^^^^^^^^
 
```
Add note to the section about changing your FST file (above) that if it is uploaded to our servers/Marketplace, they'll need to update it using the Avatar Packager.

nimisha20 replied:
Done.

jennaingersoll commented on docs/source/create/entities/avatar-interactions.md:
```diff
@@ -30,6 +38,28 @@ The following example walks you through the process of creating a simple entity,
+
+## Create Avatar Entities
+
+Your avatar will also interact with avatar entities. For example, wearables such as hats or glasses are avatar entities. Avatar entities are attached to your avatar, and unlike domain entities, they travel with your avatar when you go to other domains. 
```
Recommendation: Define first, then give examples :)
```suggestion
Your avatar will also interact with avatar entities. Avatar entities are entities that are attached to your avatar, and unlike domain entities, they travel with your avatar when you go to other domains. Examples of avatar entities include wearables such as glasses or hats.
```

jennaingersoll commented on docs/source/create/entities/avatar-interactions.md:
```diff
@@ -30,6 +38,28 @@ The following example walks you through the process of creating a simple entity,
+Avatar entities can travel with your avatar because they exist on the [Avatar Mixer](../../explore/get-started/architecture.html#assignment-clients). We've listed the ways you can create avatar entities with some examples:
+
+1. [Create a wearable:](../wearables.html) All wearables are avatar entities. 
+2. [Clone as an avatar entity:](entity-behavior.html#make-an-entity-cloneable) If you don't want to give users permission to create or edit entities in your domain, you can set an entity to be cloned as an avatar entity. For example, if you have a coffee shop in your domain, you can set all coffee cups to be cloned as avatar entities. Any user who clones a coffee cup is creating an avatar entity  ...
```
```suggestion
2. [Clone as an avatar entity:](entity-behavior.html#make-an-entity-cloneable) When you clone an entity as an avatar entity, you make a copy of the entity and attach it to your avatar. Every copy of that entity will now leave with the avatar when they leave the domain. For example, if you have a coffee shop in your domain, you can set all coffee cups to be cloned as avatar entities. Any user who clones a coffee cup will take the avatar entity with them when they exit the domain. You can keep your domain free of clutter using this property.
```

jennaingersoll commented on docs/source/host/run-domain-docker.rst:
```diff
@@ -0,0 +1,121 @@
+   This is the tag and needs to be updated with the release of the latest version of High Fidelity. 
+3. Next, you'll need to add information about your domain in the 'hifi.yml'.
+
+::
```
Can we indent this code sample?

nimisha20 replied:
Done.

jennaingersoll commented on docs/source/host/run-domain-docker.rst:
```diff
@@ -0,0 +1,121 @@
+
+5. Go to the directory in the command line tool and run the image:
+
+::
```
Pretty sure the :: doesn't have to be on its own line. Instead you can add it to the previous one.

```suggestion
5. Go to the directory in the command line tool and run the image::
```

nimisha20 replied:
Done

jennaingersoll commented on docs/source/host/run-domain-docker.rst:
```diff
@@ -0,0 +1,121 @@
+
+6. To stop running your container, run the following in the command line tool:
+
+``docker-compose stop``
```
Let's put this line with a block syntax so it matches all of the others ::

nimisha20 replied:
Done

jennaingersoll commented on docs/source/host/run-domain-docker.rst:
```diff
@@ -0,0 +1,121 @@
+Update Your Container to the Latest Version
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can also update your container when a new version is released by changing the version number in the .env file.
```
```suggestion
You can update your container when a new version is released by changing the version number in the .env file.
```

jennaingersoll commented on docs/source/host/run-domain-docker.rst:
```diff
@@ -0,0 +1,121 @@
+
+**See Also**
+
++ :doc:`Host Your Domain`
```
```suggestion
+ :doc:`Host Your Domain<host-your-domain>`
```

jennaingersoll commented on docs/source/host/run-domain-docker.rst:
```diff
@@ -0,0 +1,121 @@
+**See Also**
+
++ :doc:`Host Your Domain`
++ :doc:`Set Up Your Domain`
```
```suggestion
+ :doc:`Set Up Your Domain<your-domain>`
```

jennaingersoll commented on docs/source/host/domain-settings-event.rst:
```diff
@@ -0,0 +1,33 @@
+
+We've listed some guidelines you can follow when you set up your domain for an event. 
+
+.. note:: Keep in mind that domain settings will vary depending on your requirements. We've listed the best practices, you can tailor these to suit your needs.
```
```suggestion
.. note:: Keep in mind that domain settings will vary depending on your requirements. Here, we've listed the best practices, and you can tailor these to suit your needs.
```

jennaingersoll commented on docs/source/host/domain-settings-event.rst:
```diff
@@ -0,0 +1,33 @@
+.. note:: Keep in mind that domain settings will vary depending on your requirements. We've listed the best practices, you can tailor these to suit your needs.
+
+
+1. `Host your domain <host-your-domain.html>`_: If you're expecting a large number of users, we recommend hosting your domain on a virtual machine. You'll need to determine the size of the virtual machine you want to host your domain on. This varies depending on the number of users you're expecting and the content in your domain. 
```
```suggestion
1. :doc:`Host your domain <host-your-domain>`: If you're expecting a large number of users, we recommend hosting your domain on a virtual machine. You'll need to determine the size of the virtual machine you want to host your domain on. This varies depending on the number of users you're expecting and the content in your domain. 
```

jennaingersoll commented on docs/source/host/domain-settings-event.rst:
```diff
@@ -0,0 +1,33 @@
+
+
+1. `Host your domain <host-your-domain.html>`_: If you're expecting a large number of users, we recommend hosting your domain on a virtual machine. You'll need to determine the size of the virtual machine you want to host your domain on. This varies depending on the number of users you're expecting and the content in your domain. 
+2. `Set Up User Permissions <secure-domain.html#set-user-permissions>`_: It is good practice to set up user permissions and create entity filters so that users cannot create or edit entities in your domain. You can keep your domain clutter free and keep users from modifying the existing domain content. 
```
```suggestion
2. `Set Up User Permissions <secure-domain.html#set-user-permissions>`_: It is good practice to set up user permissions and create entity filters so that users cannot create or edit entities in your domain. You can keep your domain clutter-free and keep users from modifying the existing domain content. 
```

jennaingersoll commented on docs/source/host/domain-settings-event.rst:
```diff
@@ -0,0 +1,33 @@
+
+1. `Host your domain <host-your-domain.html>`_: If you're expecting a large number of users, we recommend hosting your domain on a virtual machine. You'll need to determine the size of the virtual machine you want to host your domain on. This varies depending on the number of users you're expecting and the content in your domain. 
+2. `Set Up User Permissions <secure-domain.html#set-user-permissions>`_: It is good practice to set up user permissions and create entity filters so that users cannot create or edit entities in your domain. You can keep your domain clutter free and keep users from modifying the existing domain content. 
+3. Lock down avatar size: Domains can have loading issues if avatars keep changing their sizes. When you lock down an avatar size, you are ensuring that users cannot affect your domain by changing their avatar size beyond a limit. 
```
Maybe something about why this happens? Because your domain is reloading the content from the avatar when they scale up or down?

nimisha20 replied:
Done.

nimisha20 replied:
This is to avoid avatars from breaching some settings. Ryan advised me to skip this point. Removed.

jennaingersoll commented on docs/source/host/domain-settings-event.rst:
```diff
@@ -0,0 +1,33 @@
+1. `Host your domain <host-your-domain.html>`_: If you're expecting a large number of users, we recommend hosting your domain on a virtual machine. You'll need to determine the size of the virtual machine you want to host your domain on. This varies depending on the number of users you're expecting and the content in your domain. 
+2. `Set Up User Permissions <secure-domain.html#set-user-permissions>`_: It is good practice to set up user permissions and create entity filters so that users cannot create or edit entities in your domain. You can keep your domain clutter free and keep users from modifying the existing domain content. 
+3. Lock down avatar size: Domains can have loading issues if avatars keep changing their sizes. When you lock down an avatar size, you are ensuring that users cannot affect your domain by changing their avatar size beyond a limit. 
+4. Create audio attenuation zones: Depending on the type of event you're hosting, you will need to create audio attenuation zones. These zones control how users hear audio. For example, if you're having an event with two hosts on stage, the hosts and audience members should hear the hosts louder than everyone else. You can set up two attenuation zones, one for the stage and the other for the audi ...
```
I think we have docs somewhere for attenuation. We should link to those.

nimisha20 replied:
Done.

jennaingersoll commented on docs/source/host/domain-settings-event.rst:
```diff
@@ -0,0 +1,33 @@
+4. Create audio attenuation zones: Depending on the type of event you're hosting, you will need to create audio attenuation zones. These zones control how users hear audio. For example, if you're having an event with two hosts on stage, the hosts and audience members should hear the hosts louder than everyone else. You can set up two attenuation zones, one for the stage and the other for the audi ...
+5. Whitelist scripts: This step ensures that only whitelisted scripts can run in the domain. You are avoiding risking running any unauthorized scripts that could affect your domain and any users present.
+6. Use only `server scripts <../script/get-started-with-scripting.html#types-of-scripts>`_: Local scripts can slow down load times for a domain. Use server scripts instead.
+7. `Bake your content <manage-assets.html#bake-an-asset>`_: Loading models in High Fidelity involves real-time rendering (drawcalls), which means images are produced and analyzed in real-time. Depending on the complexity of the models, the drawcalls can differ. You can reduce the drawcalls and optimize the rendering process using baking.
```
Note: the link to this may change with the oven docs you're currently writing.

nimisha20 replied:
It does! Done.

jennaingersoll commented on docs/source/host/domain-settings-event.rst:
```diff
@@ -0,0 +1,33 @@
++ `Set Up User Permissions <secure-domain.html#set-user-permissions>`_
++ :doc:`Configure Your Domain Settings <your-domain/configure-settings>`
++ :doc:`Manage Your Assets <manage-assets>`
++ :doc:`Tutorial: Set Up A Theater Domain <theater-domain-tutorial>`
```
Remove this one. We're going to remove/redo this tutorial in 83, so might as well not add it, then have to remove it next month.

nimisha20 replied:
Done.

jennaingersoll commented on docs/source/faq.rst:
```diff
@@ -5,7 +5,7 @@ Frequently Asked Questions
 We get a lot of questions from our users! If you have questions about our product, feel free to browse through this page to learn more. And if you don't see your answer here, e-mail us at `support@highfidelity.com <mailto:support@highfidelity.com>`_!
 
```
Add the mini-TOC here now that we have multiple sections.

nimisha20 replied:
Done.

jennaingersoll commented on docs/source/faq.rst:
```diff
@@ -16,98 +16,169 @@ Marketplace
+|                                                                                                     |
+| Can I rez multiple copies of a Marketplace item?                                                    |
++=====================================================================================================+
+| No. You will need to buy multiple copies of the item. Marketplace items behave like items in the    |
```
Unless the seller has allowed it to be rezz'd unlimited times in a single domain. (This was added in 81)

https://docs.highfidelity.com/en/rc81/sell/add-item.html#unlimited-rezzing-of-your-item

nimisha20 replied:
Done

jennaingersoll commented on docs/source/explore/socialize.rst:
```diff
@@ -80,10 +80,10 @@ As an administrator in a domain, you will have privileges to maintain a domain.
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can enable a privacy bubble that protects your personal space in the metaverse. When it is enabled, other people will disappear if they get too close to you. Your privacy bubble is disabled by default. To enable it, pull up your tablet or HUD and click **Bubble**. In Desktop mode, you can also use the keyboard shortcut ``CTRL`` + ``N``.
+You can enable a privacy shield that protects your personal space in the metaverse. When it is enabled, other people will disappear if they get too close to you. Your privacy shield is disabled by default. To enable it, pull up your tablet or HUD and click **Shield**. In Desktop mode, you can also use the keyboard shortcut ``CTRL`` + ``N``.
```
We have a new icon next to the audio bar in the top-left corner of interface. We should mention this.

nimisha20 replied:
Done.

jennaingersoll commented on docs/source/explore/get-started/vr-controls.rst:
```diff
@@ -15,15 +15,114 @@ Use these controls on your VR controllers so that you can enjoy your experience
 
 .. note:: If you want to jump and fly in VR, you must enable it under Settings > Controls > VR Movement.
```
```suggestion
.. note:: If you want to jump, fly, or strafe (walk sideways) in VR, you must enable it under Settings > Controls > VR Movement.
```

jennaingersoll commented on docs/source/explore/get-started/vr-controls.rst:
```diff
@@ -15,15 +15,114 @@ Use these controls on your VR controllers so that you can enjoy your experience
++----------------------------+---------------------------------------------------------------------------------+
+| *Control Scheme Selection:*| In Default mode, your walking speed doesn't change depending on how far forward |
+| Default                    | you push your controller's joystick. Fully pushing your joystick forward will   |
+|                            | make your avatar run at twice your walking speed.                               |
```
```suggestion
|                            | make your avatar run.                               |
```

I don't think its important to mention that running is twice your walking speed. I think most people think of run as "faster than walking" so its needless to say what the technical detail of that is. Plus, once you get into Analog and Analog++, what does twice the walking speed mean since your walkign speed varies based on how far forward you push the joystick. So we'll stick with "walk or run".

jennaingersoll commented on docs/source/explore/get-started/vr-controls.rst:
```diff
@@ -15,15 +15,114 @@ Use these controls on your VR controllers so that you can enjoy your experience
++----------------------------+---------------------------------------------------------------------------------+
+| *Control Scheme Selection:*| In Analog mode, your walking speed changes based on how far forward you push    |
+| Analog                     | your controller's joystick. Fully pushing your joystick forward will            |
+|                            | make your avatar run at twice your walking speed.                               |
```
```suggestion
|                            | make your avatar run.                           |
```

jennaingersoll commented on docs/source/explore/get-started/vr-controls.rst:
```diff
@@ -15,15 +15,114 @@ Use these controls on your VR controllers so that you can enjoy your experience
+| *Control Scheme Selection:*| In Analog++ mode, your walking speed changes based on how far forward you push  |
+| Analog++                   | your controller's joystick. You can also use the slider to change the walking   |
+|                            | speed in meters/second. Fully pushing your joystick forward will make your      |
+|                            | avatar run at twice your walking speed.                                         |
```
```suggestion
|                            | avatar run.                                      |
```

jennaingersoll commented on docs/source/explore/get-started/vr-controls.rst:
```diff
@@ -15,15 +15,114 @@ Use these controls on your VR controllers so that you can enjoy your experience
 Gamepad
 -------------------------
 
 While the use of a gamepad is supported, High Fidelity is best experienced with hand controllers like the Oculus Touch, HTC Vive, and Microsoft MR controllers.
```
```suggestion
While the use of a gamepad is supported, High Fidelity is best experienced with VR equipment or the keyboard in Desktop mode.
```

jennaingersoll commented on docs/source/explore/get-started/vr-controls.rst:
```diff
@@ -15,15 +15,114 @@ Use these controls on your VR controllers so that you can enjoy your experience
++----------------------------+---------------------------------------------------------------------------------+
 
 -------------------------
 Gamepad
```
Does Gamepad belong in the Desktop page? I feel like its not likely to be used with an HMD, but rather with a computer...

nimisha20 replied:
This seems to be something no one is really agreeing on - gamepads are also used for VR experiences. I'm going to leave it as it is and create a manuscript case for this.

jennaingersoll commented on docs/source/explore/get-started/vr-controls.rst:
```diff
@@ -15,15 +15,114 @@ Use these controls on your VR controllers so that you can enjoy your experience
 
```
Since we're focusing less on HMD and more on Desktop, we should remove any reference to "VR equipment is better than Desktop". So change the first paragraph to something more vague like 

```suggestion
To interact with people in 3D and track body movements, you will need to use VR equipment such as the Oculus Rift or HTC Vive. With VR equipment, you can have a fully immersive experience in the 3D metaverse.
```

nimisha20 replied:
Should we use the word metaverse? I feel like we're shifting away from what it means to a lot of users. Can we replace that with just High Fidelity instead?

nimisha20 replied:
I modified this line and made it High Fidelity instead.

jennaingersoll commented on docs/source/create/entities/avatar-interactions.md:
```diff
@@ -1,12 +1,20 @@
 # Define Interactions with Avatars
 
+Your avatar can interact with different types of entities. You can change the color of a cube entity as your avatar moves closer to it, or create an avatar entity that travels with you through different domains. 
```
```suggestion
In real life, you interact with objects on a daily basis. In High Fidelity, your avatar can also interact with objects (entities) in the metaverse. There are a number of ways you can define the interactions you have with objects: you can write scripts to change the properties of an entity. You can create entities that are unique to your avatar (we call these "avatar entities"). And don't forget that you can set an entity's behavior and collision properties, so that objects are grabbable, triggerable, or dynamic.
```

jennaingersoll commented on docs/source/create/entities/avatar-interactions.md:
```diff
@@ -30,6 +38,28 @@ The following example walks you through the process of creating a simple entity,
+
+Your avatar will also interact with avatar entities. For example, wearables such as hats or glasses are avatar entities. Avatar entities are attached to your avatar, and unlike domain entities, they travel with your avatar when you go to other domains. 
+
+Avatar entities can travel with your avatar because they exist on the [Avatar Mixer](../../explore/get-started/architecture.html#assignment-clients). We've listed the ways you can create avatar entities with some examples:
```
```suggestion
Avatar entities live on the [Avatar Mixer](../../explore/get-started/architecture.html#assignment-clients), so they are connected to (and move with) your avatar. We've listed the ways you can create avatar entities with some examples:
```

jennaingersoll commented on docs/source/create/entities/create-fountain.rst:
```diff
@@ -0,0 +1,93 @@
+Tutorial: Create a Smoke Fountain
+#####################################
+
+In this tutorial, you will learn how particle entities work by creating a smoke fountain that emits multiple colors. Particle entities are used to create effects made up of many small particles, such as fire, smoke, or falling leaves.
```
Fire and smoke are similar concepts. So let's replace with something different :)

```suggestion
In this tutorial, you will learn how particle entities work by creating a smoke fountain that emits multiple colors. Particle entities are used to create effects made up of many small particles, such as smoke, confetti, or falling leaves.
```

jennaingersoll commented on docs/source/create/entities/create-fountain.rst:
```diff
@@ -0,0 +1,93 @@
+Create a Smoke Fountain
+---------------------------------
+
+Particle entities are used to create effects that are made up of smaller parts such as smoke, fire, or swirling water. You can create any effect of your choice using different textures. The default texture for all particle entities is a `wispy smoke texture <https://content.highfidelity.com/DomainContent/production/Particles/wispy-smoke.png>`_. You can edit other particle entity properties or rep ...
```
Took out the last sentence b/c i think we support a lot more textures now.

```suggestion
Particle entities are used to create effects that are made up of smaller parts such as smoke, confetti, or falling leaves. The entity's effect and appearance is defined by its texture. The default texture is a `wispy smoke texture <https://content.highfidelity.com/DomainContent/production/Particles/wispy-smoke.png>`_, but you can replace this texture with your own to create your desired effect. 
```

nimisha20 replied:
I'll include the new types of supported textures to the PBR Materials page as well.

jennaingersoll commented on docs/source/create/entities/create-fountain.rst:
```diff
@@ -0,0 +1,93 @@
+
+Particle entities are used to create effects that are made up of smaller parts such as smoke, fire, or swirling water. You can create any effect of your choice using different textures. The default texture for all particle entities is a `wispy smoke texture <https://content.highfidelity.com/DomainContent/production/Particles/wispy-smoke.png>`_. You can edit other particle entity properties or rep ...
+
+In this tutorial, you will create this smoke fountain by editing a few properties of particle entities. This is to help you get familiar with how changing these properties changes how the entity looks and behaves. 
```
Remove this -- we already said it in the intro what we're doing 

```suggestion
To create your smoke fountain using a particle entity: 
```

jennaingersoll commented on docs/source/create/entities/create-fountain.rst:
```diff
@@ -0,0 +1,93 @@
+
+1. In Interface, pull up your HUD or Tablet and go to **Create**.
+2. Click the 'Particle' icon to create the particle entity. By default, the particle entity emits smoke. 
+3. In the 'Properties' tab, change the values of the following properties:
```
```suggestion
3. Go to the 'Properties' tab, and set the following values:
```

jennaingersoll commented on docs/source/create/entities/create-fountain.rst:
```diff
@@ -0,0 +1,93 @@
+
+.. image:: _images/smoke-fountain.gif
+
+Congratulations! You've created a multi-colored smoke fountain!
```
```suggestion
Congratulations! You've created a multi-colored smoke fountain! You can experiment with different particle settings to simulate movement, such as a waterfall, confetti gun or falling leaves. 
```

nimisha20 replied:
Made a minor change to this suggestion.

jennaingersoll commented on docs/source/create/entities/create-fountain.rst:
```diff
@@ -0,0 +1,93 @@
+
+Congratulations! You've created a multi-colored smoke fountain!
+
+You can also try creating some of these effects using particle entities:
```
```suggestion
```

jennaingersoll commented on docs/source/create/entities/create-fountain.rst:
```diff
@@ -0,0 +1,93 @@
+You can also try creating some of these effects using particle entities:
+
+
++ Fire pit
```
```suggestion
```

jennaingersoll commented on docs/source/create/entities/create-fountain.rst:
```diff
@@ -0,0 +1,93 @@
+
+
++ Fire pit
++ Waterfall
```
```suggestion
```

jennaingersoll commented on docs/source/create/entities/create-fountain.rst:
```diff
@@ -0,0 +1,93 @@
+
+- :doc: `Create New Entities <create-entities.html>`
+- :doc: `Change How Entities Look <entity-appearance.html>`
+- :doc: `Creator Tools <../tools.html>`
```
```suggestion
- :doc: `Create Tools <../tools.html>`
```

nimisha20 committed:
21113: Add Documentation for the Oven Tool to Docs page

jennaingersoll commented on docs/source/host/oven.rst:
```diff
@@ -0,0 +1,250 @@
+###################################
+
+You can reduce your load times by optimizing your domain content, models, or avatar using Oven, a standalone application created by High Fidelity. This process of optimizing content is called baking. 
+
```
Add TOC

nimisha20 replied:
Done.

jennaingersoll commented on docs/source/host/oven.rst:
```diff
@@ -0,0 +1,250 @@
+.. image:: _images/oven.png
+
+
+^^^^^^^^^^^^^^^^^^^^^^
```
I'd leave all of these as 

```suggestion
----------------------------
Bake Your Domain
----------------------------
```

nimisha20 commented on docs/source/create/3d-models/pbr-materials-guide.rst:
```diff
@@ -74,6 +74,7 @@ Notes:
 1. If you set transparency with a texture, the transparency (alpha) should be in the material’s albedo texture, as a PNG file with transparency and not as a separate transparency texture.
 2. High Fidelity’s renderer can draw two different kinds of transparency: "alpha" (255 graduates steps of transparency, no shading on surface, casts no shadows,) and "mask" (binary transparency, full shading of opaque surface, whole surface casts shadow.)
 3. To determine whether a texture is treated as a mask or as alpha, the engine looks for alpha values between 2% and 98%. An easy way to create a mask texture is to save your image as a PNG-8 since it only supports binary transparency, while PNG-24 supports a range of transparency levels.
+4. Currently, we support using a second UV set with the following texture channels only - Light map and Ambient Occlusion. 
```
```suggestion
4. We support using a second UV set with the following texture channels only - Light map and Ambient Occlusion. 
```

nimisha20 commented on docs/source/create/avatars/custom-animations.rst:
```diff
@@ -171,26 +193,24 @@ We've listed the animation roles and their description. Animation roles are freq
-4. Under 'Avatar animation JSON', add the URL for your Animation Graph  file.  ![](avatar-settings.png)
+4. Under 'Avatar animation JSON', paste the URL for your JSON file. 
+
+.. image:: _images/avatar-anim.png
```
```suggestion

```

nimisha20 commented on docs/source/create/entities/create-fountain.rst:
```diff
@@ -0,0 +1,93 @@
+
+.. image:: _images/smoke-fountain.gif
+
+Congratulations! You've created a multi-colored smoke fountain!
```
```suggestion
Congratulations! You've created a multi-colored smoke fountain! You can experiment with different settings to simulate particle movement, such as a waterfall, confetti gun or falling leaves. 
```

nimisha20 commented on docs/source/host/domain-settings-event.rst:
```diff
@@ -0,0 +1,33 @@
++ `Set Up User Permissions <secure-domain.html#set-user-permissions>`_
++ :doc:`Configure Your Domain Settings <your-domain/configure-settings>`
++ :doc:`Manage Your Assets <manage-assets>`
++ :doc:`Tutorial: Set Up A Theater Domain <theater-domain-tutorial>`
```
```suggestion

```

nimisha20 committed:
Apply suggestions from code review

Committed most suggestions, adding the rest of the changes through other commits.

Co-Authored-By: nimisha20 <nimisha@highfidelity.io>

nimisha20 committed:
Changes from review.
